tools/pylib/_boutpp_build/backend.py

```diff
@@ -166,6 +173,10 @@ def build_sdist(sdist_directory, config_settings=None):
                     enable_xz = True
                 else:
                     raise ValueError(f"unknown option {v} for {k}")
+            if k == "dist":
```

Comment:
Might make sense to use something like `argparse` for this in a future version?

Reply:
probably, but I was lazy :see_no_evil: